MobiMart/customer/views.py

```diff
@@ -30,6 +55,11 @@ def register(request):
             elif User.objects.filter(email=email).exists():
                 messages.info(request,"Email Already Exists")
                 return redirect("register")
+            elif not checkpass(passw1):
```

Comment:
Here you are calling the checkpass, but the checkpass is not returning anything.